nvflare/edge/device/android/sdk/AndroidExecutor.kt

```diff
@@ -0,0 +1,93 @@
+package com.nvidia.nvflare.sdk
+
+import android.util.Log
+import com.nvidia.nvflare.sdk.core.DXO
+import com.nvidia.nvflare.sdk.core.Context
+import com.nvidia.nvflare.sdk.core.Signal
+import com.nvidia.nvflare.sdk.core.Executor
+import com.nvidia.nvflare.sdk.training.Trainer
+import com.nvidia.nvflare.sdk.training.ETTrainer
+import com.nvidia.nvflare.sdk.models.TrainingConfig
+import kotlinx.coroutines.runBlocking
+
+/**
+ * Android-specific implementation of Executor.
+ * Bridges the old Trainer interface with the new SDK architecture.
+ */
+class AndroidExecutor(
```

Comment:
we can call this ETTrainerExecutor or something similar, and only use ET here, forget about that Trainer interface we have in the previous (temp) version.